cmd/api/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"log"
+
+	_ "example.com/tfgrid-kyc-service/api/docs"
+	"example.com/tfgrid-kyc-service/internal/configs"
+	"example.com/tfgrid-kyc-service/internal/server"
+)
+
+//	@title			TFGrid KYC API
+//	@version		0.1.0
+//	@description	This is a KYC service for TFGrid.
+//	@termsOfService	http://swagger.io/terms/
+
+// @contact.name	Codescalers Egypt
+// @contact.url	https://codescalers-egypt.com
+// @contact.email	info@codescalers.com
+// @BasePath		/
+func main() {
+	config, err := configs.LoadConfig()
+	if err != nil {
+		log.Fatal("Failed to load configuration:", err)
```

Comment:
error messages should be lower case and no punctuation in the end and in case of static error messages extract them into global errors

Reply:
typically in main should be just 

```
func main(){
   // params needed to init the app
   app = internal.NewApp( .... ) 
    if err:=app.Run(); err!=nil {
         log.Fatal(...)
    }

}
```

Reply:
> > error messages should be lower case and no punctuation in the end and in case of static error messages extract them into global errors
> 
> You are referring to log messages here. Can you confirm that you want me to apply the lowercase rule to log messages? For log messages, I start with a capital letter, but when I create an error or return one, I use lowercase unless something slips through :)

Yes, all should be lowered

Reply:
> and then pass it all to the server constructor. I prefer to do this here.
> Do you want me to move this to the server constructor instead?

I added a comment, basically your code had some Fatal/panic/..etc and these shouldn't happen anywhere except in the main func, as your code needs to have 1 single point of exit.